ckan/controllers/organization.py

```diff
@@ -1,19 +1,23 @@
+import re
+
 import ckan.controllers.group as group
 
 
 class OrganizationController(group.GroupController):
-    ''' The organization controller is pretty much just the group
-    controller. It has a few templates defined that are different and sets
-    the group_type to organization so that the group controller knows that
-    it is in fact the organization controller.  All the main logical
-    differences are therefore in the group controller.
+    ''' The organization controller is for Groups of type 'organization'. It
```

Comment:
`''' The organization controller is for Organizations, which are implemented as ckan groups with is_organization=True`